shared/lib_battery_dispatch_automatic_btm.h

```diff
@@ -80,7 +80,8 @@ class dispatch_automatic_behind_the_meter_t : public dispatch_automatic_t
         std::vector<double> battCycleCost,
         double interconnection_limit,
         bool chargeOnlySystemExceedLoad,
-        bool dischargeOnlyLoadExceedSystem
+        bool dischargeOnlyLoadExceedSystem,
+        double SOC_min_outage
```

Comment:
at some point we should stick with one naming convention and c++ instead of java structure. Just a comment and nothing specific to this pull request